contracts/extensions/TokenSupplier.sol

```diff
@@ -39,6 +39,14 @@ contract TokenSupplier is ColonyExtension {
   uint256 tokenIssuanceRate;
   uint256 lastIssue;
   uint256 lastRateUpdate;
+  mapping(address => uint256) metatransactionNonces;
```

Comment:
Existing extensions would have to stop inheriting from that contract.

Reply:
I was thinking about it this morning and I think that's a fair trade -- rename the current `ColonyExtension` to `ColonyExtensionNoMeta` and use that for existing extensions, and then have the new `ColonyExtension` moving forward. Assuming there are more extensions in our future it will be well worth it.

Reply:
Okay. And you think that should be in this PR?

Reply:
Mm I could go either way. I could see it in this PR but also happy to put together a quickie PR after this is merged.